bioblend/_tests/TestGalaxyQuotas.py

```diff
@@ -0,0 +1,59 @@
+"""
+Use ``nose`` to run these unit tests.
+"""
+import uuid
+
+from . import GalaxyTestBase, test_util
+
+
+@test_util.skip_unless_galaxy()
+class TestGalaxyQuotas(GalaxyTestBase.GalaxyTestBase):
+
+    def setUp(self):
+        super(TestGalaxyQuotas, self).setUp()
+        # Quota names must be unique, and they're impossible to delete
+        # without accessing the database.
+        self.quota_name = 'BioBlend-Test-Quota-%s' % uuid.uuid4().hex
+        self.quota = self.gi.quotas.create_quota(
+            self.quota_name, 'testing', '100 GB', '=',
+            default='registered'
+        )
+
+    def tearDown(self):
+        self.gi.quotas.update_quota(self.quota['id'], default='registered')
```

Comment:
This API desperately needs to be rewritten using the more modern galaxy conventions. I cannot imagine anyone scripting around this API and enjoying it.